packages/local-environment/src/network.ts

```diff
@@ -175,19 +189,20 @@ export class HardhatNetwork extends Network {
   keys: Key[];
   handler: DockerizedNetworkActor;
 
-  constructor(name: string, domain: number, options?: HardhatNetworkOptions) {
-    super(name, domain, options?.chainId || domain);
-    this.handler = options?.handler || new DockerizedNetworkActor(this.name);
+  constructor(name: string, domain: number, docker?: Dockerode, options?: HardhatNetworkOptions) {
+    super(name, domain, domain, docker);
+    this.handler = options?.handler || new DockerizedNetworkActor(this.name, this.docker);
     this.blockTime = 10;
     this.firstStart = false;
     this.keys = options?.keys || [];
+    this.rpc = [`http://localhost:${this.handler.port}`];
```

Comment:
Has to be localhost?

Reply:
Yes, all mainnet forking happens on localhost - it's just a hardhat config.